pkg/cmd/cli/describe/projectstatus_test.go

```diff
@@ -137,6 +137,8 @@ func TestProjectStatus(t *testing.T) {
 				"database deploys",
 				"frontend deploys",
 				"with docker.io/openshift/ruby-20-centos7:latest",
+				"#2 deployment failed less than a second ago: unable to contact server - 0/1 pods",
+				"#2 deployment running for 7 seconds - 2/1 pods",
 				"#1 deployed 8 seconds ago",
```

Comment:
Splitting `#n deployment` => `deployment #n` seems more readable, unless we want #n as the first thing in line. 

But I'm actually referring to line https://github.com/openshift/origin/pull/2423/files#diff-c0c5022b465088097f483637ede5dcdaR401 and the following lines.  :)